README.md

```diff
@@ -0,0 +1,49 @@
+# To Do list
+## Description
+
+> In this project, you will build a simple HTML list of To Do tasks. The list will be styled according to the specifications listed later in this lesson. This simple web page will be built using webpack and served by a webpack dev server.
+
+## Screenshot
+![image]()
+
+## Built With
+
+- HTML5, CSS3(Bootstrap 5) and JS-DOM
+
+## Live Demo
+
+
+
+
+### Prerequisites
+
+Basic knowledge of HTML Semantics CSS And JS-DOM to understand code.
+Ordinary internet browser knowledge to use. :smiley:
+
```

Comment:
* Kindly add a descriptive README file to your project it should contain the basic commands and instructions to successfully run your project locally. please use [this template](https://github.com/microverseinc/readme-template) but remember to customize it to your project

Reply:
- [ ] You made a good effort to update the README file as suggested by the reviewer. But still, there is some information left to run the project locally. Like clone the project, installing the packages, `npm run build`, `npm start` to run the project locally. Kindly try to add this instruction under the `getting started` heading.